src/Uno.UWP/Storage/Streams/DataWriterStoreOperation.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Foundation;
+
+namespace Windows.Storage.Streams
+{
+	public partial class DataWriterStoreOperation : IAsyncOperation<uint>, IAsyncInfo
+	{
+		private CancellationTokenSource _cts = new CancellationTokenSource();
+		private AsyncOperationCompletedHandler<uint> _onCompleted;
+		private Task<uint> _task;
+		private AsyncStatus _status;
+		private uint _id = 0;
```

Comment:
It seems the `_id` is not used?

Reply:
Change to public uint Id { get; set; } because IAsyncInfo needs